reana/reana_dev/utils.py

```diff
@@ -335,3 +337,123 @@ def get_prefixed_component_name(component):
     :return: Prefixed name.
     """
     return "-".join([INSTANCE_NAME, component])
+
+
+def fetch_latest_pypi_version(package):
+    """Fetch latest released version of a package."""
+    import requests
+
+    pypi_rc_info = requests.get("https://pypi.python.org/pypi/{}/json".format(package))
+    return sorted(pypi_rc_info.json()["releases"].keys())[-1]
+
+
+def is_last_commit_tagged(package):
+    """Check whether the last commit of the module points at a tag."""
+    tag = run_command("git tag --points-at", package, return_output=True)
+    return bool(tag)
+
+
+def is_feature_branch(component):
+    """Check whether component current branch is different from master."""
+    return get_current_branch(get_srcdir(component)) != "master"
+
+
+def replace_version_string(
```

Comment:
Thanks, looks great! 😊 This covers all the use cases in the create release commit. Just two observations:
- We could rename the function since this replaces strings in component files, no matter what they are
- What about renaming `module_grep` to something along the lines of: `line_selector_expression`/`line_regex`/`line_pattern` (not very gifted today with the naming but you know what I mean)

Reply:
- Mmh what about `replace_string`, `replace_regex`, `sed_file`...?
- Naming on these tricky functions is always hard, maybe `line_selector_regex`?

Reply:
Updated, feel free to change it if needed.